tooling/bin/pecl-build

```diff
@@ -3,7 +3,8 @@
 set -e
 
 rm -f ./bridge/_generated.php
-composer compile
+composer -dtooling/generation generate
+composer -dtooling/generation verify
```

Comment:
To ensure things don't get out of sync in the future, we could add `composer -dtooling/generation update` above these lines and remove the [`STEP_COMPOSER_UPDATE` step](https://github.com/DataDog/dd-trace-php/blob/1d940f3/.circleci/config.yml#L688) from the CircleCI config since that doesn't update from the correct Composer file for this anymore.

Reply:
I really like the idea. I tried to think about something that I might not consider making this change, but so far I cannot identify any CON. Change pushed. Let's see if CI stays green :).
Thanks for the idea.